src/win/dl.c

```diff
@@ -40,7 +41,8 @@ int uv_dlopen(const char* filename, uv_lib_t* lib) {
     return uv__dlerror(lib, filename, GetLastError());
   }
 
-  lib->handle = LoadLibraryExW(filename_w, NULL, LOAD_WITH_ALTERED_SEARCH_PATH);
+  flags = LOAD_WITH_ALTERED_SEARCH_PATH | LOAD_LIBRARY_SAFE_CURRENT_DIRS;
```

Comment:
```suggestion
  flags = LOAD_WITH_ALTERED_SEARCH_PATH | /*LOAD_LIBRARY_SAFE_CURRENT_DIRS*/0x2000;
```
And retry without the flag if LoadLibraryEx fails with (I assume) ERROR_INVALID_PARAMETER or ERROR_INVALID_FLAGS?